src/lib/storage/storage_manager.hpp

```diff
@@ -8,16 +8,36 @@
 #include <shared_mutex>
 #include <string>
 #include <vector>
+#include <fstream>
 
+#include "storage/chunk_encoder.hpp"
 #include "lqp_view.hpp"
 #include "prepared_plan.hpp"
 #include "types.hpp"
 
+#include "storage/dictionary_segment.hpp"
+
 namespace hyrise {
 
 class Table;
 class AbstractLQPNode;
 
+const auto MAX_CHUNK_COUNT = uint8_t{50};
```

Comment:
Not specific enough.
Maybe MAX_CHUNKS_PER_FILE?

Reply:
+1